css/base/_settings.scss

```diff
@@ -0,0 +1,19 @@
+//CHRISTMAS THEME
```

Comment:
plik _settings to nie najlepsze miejsce na trzymanie styli thema, poniewaz w tym pliku mozesz trzymac zmienne uzywane przez podstawowy wyglad strony, np nasze breakpointy

utworz osobny plik w glownym katalogu /css, np:

_christmas-theme.scss

i w nim trzymaj style thema